src/azure-cli/azure/cli/command_modules/synapse/_params.py

```diff
@@ -0,0 +1,63 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------s
+
+from azure.cli.core.commands.parameters import name_type
+
+
+# pylint: disable=too-many-statements
+def load_arguments(self, _):
+    # synapse spark
+    with self.argument_context('synapse spark') as c:
+        c.argument('workspace_name', help='The name of the workspace.')
+        c.argument('spark_pool_name', help='The name of the spark pool.')
+        c.argument('from_index', help='Optional parameter specifying which index the list should begin from.')
+        c.argument('detailed',
+                   help='Optional query parameter specifying whether detailed response is returned beyond plain livy.')
+        c.argument('size',
+                   help='The size of the returned list.By default it is 20 and that is the maximum.')
+        c.argument('tags', help='The tag of spark batch job.')
```

Comment:
use tags_type

Reply:
> use tags_type

Could you please help us understand why need to use tags_type? our sdk needs tags instead of tags_type.

Reply:
> use tags_type

fixed